flink-staging/flink-ml/src/main/scala/org/apache/flink/ml/regression/MultipleLinearRegression.scala

```diff
@@ -107,6 +107,11 @@ class MultipleLinearRegression extends Predictor[MultipleLinearRegression] {
     this
   }
 
+  def setOptimizationMethod(optimizationMethod: String): this.type = {
```

Comment:
@chiwanpark, that could also be a solution. Then you could create an instance of a sub class with the respective parameters (e.g. the decay value). That makes also more sense, since not all optimization methods depend on the decay value. I've opened a PR where I did something similar for the abstraction of the actual calculation scheme (https://github.com/rawkintrevo/flink/pull/1). This should be easy to extend to include the decay value only for the `Xu` and `InvScaling` methods. Then we can remove the decay parameter from `IterativeSolver`.